jekyll/_posts/2022-02-04-nim-devroom-fosdem-2022.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Nim Devroom at FOSDEM 2022"
+author: pietroppeter, pmunch, dom96
+excerpt: "Tomorrow (Sat, Feb 5th) there is a Nim Devroom at FOSDEM 2022"
+---
+
+## Nim Devroom at FOSDEM 2022
+
+We are happy to announce that all preparations for the Nim Devroom at FOSDEM 2022 are now complete and
+we are looking forward to 8 excellent talks with more than 5 hours of Nim-related content. The conference is online this year
+and the talks will stream live tomorrow on the FOSDEM website. This is the final schedule for the Nim Devroom, which will be
+held tomorrow (Saturday, February 5th) from 12:30 to 17:40: https://fosdem.org/2022/schedule/track/nim_programming_language/
+
+**Note that all times are Brussels' times (CET).**
+
+FOSDEM is a two-day event organised by volunteers to promote the widespread use of free and open-source software.
+It normally takes place in Brussels, but this year (like last year) it will be held remotely. Participation is open to anyone
+and there is no need to register. This year is the first year that Nim has a dedicated devroom and we want to send special
+thanks to our speakers for their contributions!
+
+You will be able to interact with the speakers through a Matrix chat and at the end of every talk there will be a live Q&A with
+the speaker. You may use a Matrix Client ([Element](https://element.io) is good choice that you can use both from web and as an
+app) and use the link provided in the page of any talk, or you can access the chat at this link:
+https://chat.fosdem.org/#/room/#nim-devroom:fosdem.org
+
+Are you ready to:
+
+* Hear the [past, present and future of concurrency](https://fosdem.org/2022/schedule/event/nim_concurrency/)
+  in Nim directly from the mouth of our BDFL
```

Comment:
"From the mouth" reads a little odd

```suggestion
  in Nim directly from our BDFL
```

Reply:
Actually, thinking about it some more. Maybe instead of "BDFL" just write "Nim's creator". BDFL is a tounge in cheek thing and probably shouldn't be referred to in official posts like this :)

Reply:
I will go with creator